hummingbot/client/command/gateway_command.py

```diff
@@ -319,7 +311,7 @@ async def _update_gateway_configuration(self, key: str, value: Any):
         except Exception:
             self.notify("\nError: Gateway configuration update failed. See log file for more details.")
 
-    async def _show_gateway_configuration(self, key: str):
+    async def _show_gateway_configuration(self, key: str = None):
```

Comment:
The same comment here for the `key` parameter

Reply:
Changed!